1] guessing game/Hadaci hra cislo.py

```diff
@@ -0,0 +1,21 @@
+import random
+def guess_the_number():
+    secret_number = random.randint(1, 100)
+
+    while True:
+        try:
```

Comment:
ten try je prilis sirokej a lepsi to omezit jen na ten kus kodu kde cekas ze to vyhdi vyjimku